#! /bin/sh -e

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"         
    dir="$3/"            
elif [ $# -ne 1 ]; then  
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1           
fi               
case "$1" in         
    -patch)          
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

+++ ogminfo.c
@@ -188,7 +188,7 @@
     }
     vorbis_comment_clear(&vc);
   } else if ((stream->stype == 'a') && stream->vorbis) {
-    if (verbose < 1)
+    if (verbose < 2)
       return;
     fprintf(stdout, "(%s) a%d: % 7ld bytes granulepos: % 10lld pno: % 10lld ",
              __FILE__, stream->sno, pack->bytes, ogg_page_granulepos(page),
